yarn.lock

```diff
@@ -14278,8 +14328,9 @@ typescript@3.6.2:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.6.2.tgz#105b0f1934119dde543ac8eb71af3a91009efe54"
 
 typescript@^3.6.4:
-  version "3.6.4"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.6.4.tgz#b18752bb3792bc1a0281335f7f6ebf1bbfc5b91d"
+  version "3.8.3"
```

Comment:
upgraded typescript to support `declaration` and `allowJs` options at the same time https://github.com/microsoft/TypeScript/issues/7546